.nengobones.yml

```diff
@@ -1,9 +1,111 @@
+project_name: Nengo Loihi
 pkg_name: nengo_loihi
 repo_name: nengo/nengo-loihi
 
+copyright_start: 2018
+description: Run Nengo models on the Loihi chip
+
+license_rst: {}
+
+contributing_rst: {}
+
+contributors_rst: {}
+
+manifest_in:
+  global-exclude:
+    - "*.pkl"
+  recursive-include:
+    - nengo_loihi *.c
+    - nengo_loihi *.h
+    - nengo_loihi *.template
+  include:
+    - pytest.ini
+  exclude:
+    - .ci/travis_rsa.enc
+
+setup_py:
+  python_requires: ">=3.4"
```

Comment:
Should this be pegged to 3.5.2?

Reply:
I think we want to allow people to use nengo-loihi who are only using it for the emulator, in which case there's no need to pin it to 3.5.2.  We should probably add some TravisCI jobs to cover other versions though, with that in mind.